priv/graphql/schemas/admin/muc_light.gql

```diff
@@ -7,10 +7,10 @@ type MUCLightAdminMutation @protected{
     @protected(type: DOMAIN, args: ["mucDomain"])
   "Change configuration of a MUC Light room"
   changeRoomConfiguration(room: JID!, owner: JID!, name: String!, subject: String!): Room
-    @protected(type: DOMAIN, args: ["room", "owner"])
```

Comment:
I think it's better to have more than fewer variables from which we take the domain. That way we are safe against some edge cases when the room has an owner from a different domain